Cargo.toml

```diff
@@ -39,3 +39,4 @@ fs_extra = "1.1.0"
 
 [features]
 vendored-openssl = ['openssl/vendored']
+slow_tests = []
```

Comment:
what is this for?

Reply:
The test for rust is very slow, so I wanted to exclude it when running locally. It's enabled in the CI.